tests/kontonummer.js

```diff
@@ -0,0 +1,195 @@
+// external
+const chai = require('chai');
+
+// internal
+const kontonummer = require('../kontonummer');
+
+describe('Kontonummer suite', () => {
+  describe('fillZeros()', () => {
+    const mockBank = {
+      name    : 'Swedbank',
+      regex   : /^(8[0-9]{4})(0*[0-9]+)$/,
+      range   : '8000-8999',
+      modulo  : 10,
+      lengths : {
+        clearing : 5,
+        account  : 10,
+        control  : 10
+      },
+      zerofill: true
+    };
+    const fakeAccountNumbers = [
+      '8123412345',
+      '899991',
+      '854321234567890'
+    ];
+    const expectations = [
+      '812340000012345',
+      '899990000000001',
+      '854321234567890'
+    ];
+
+    it('should correctly prepend zeros to account number(s)', () => {
+      fakeAccountNumbers.forEach((accountNumber, i) => {
+        const result = kontonummer._fillZeros(accountNumber, mockBank);
+
+        chai.assert.strictEqual(expectations[i], result);
+      });
+    });
+  });
+
+  describe('getCtrlNum()', () => {
+    it('should correctly return the control number based on property `lengths.control`', () => {
+      const mockControlLength = 10;
+
+      const expected = '2345678901';
+      const result = kontonummer._getCtrlNum(mockControlLength, '12345678901');
+
+      chai.assert.strictEqual(expected, result);
+    });
+
+    it('should correctly return the control number based on property `lengths.control`', () => {
+      const mockControlLength = 11;
+
+      const expected = '12345678901';
+      const result = kontonummer._getCtrlNum(mockControlLength, '12345678901');
+
+      chai.assert.strictEqual(expected, result);
+    });
+  });
+
+  describe('getClearingNumber()', () => {
+    const mockClearingLength = 4;
+    const fakeAccountNumbers = [
+      '91801234567890',
+      '91811234567890',
+      '91891234567890',
```

Comment:
I'm still wondering:

> What's the reason for testing multiple account numbers here that match the same bank?

https://github.com/Advisa/kontonummer.js/pull/2#discussion_r204045062

Reply:
I'm not entirely sure what my thought was, but I think I wanted to be more verbose on what the input/output are to demonstrate more than just one example